github/HookDescription.py

```diff
@@ -27,11 +27,12 @@
 # along with PyGithub. If not, see <http://www.gnu.org/licenses/>.             #
 #                                                                              #
 ################################################################################
+from __future__ import annotations
 
-import github.GithubObject
+from github.GithubObject import Attribute, NonCompletableGithubObject, NotSet
 
 
-class HookDescription(github.GithubObject.NonCompletableGithubObject):
+class HookDescription(NonCompletableGithubObject):
     """
     This class represents HookDescriptions
     """
```

Comment:
```suggestion
class HookDescription(NonCompletableGithubObject):
    """
    This class represents HookDescriptions
    """

    _events: Attribute[list[str]]
    _name: Attribute[str]
    _schema: Attribute[list[list[str]]]
    _supported_events: Attribute[list[str]]
```

Reply:
It's in the method initAttributes

Reply:
It should be in one place for all classes: Either at the beginning of the class, or in the `_initAttributes`. Most classes in this PR do the former, some the latter.

Maybe we move the `_initAttributes` up to be the first method in a class, then we have both: defining types in `_initAttributes` and defining attributes at the beginning (in the first method) of the class.

Reply:
move _initAttributes to first looks like a good idea

Reply:
Now we have two ways of typing the attributes, can we stick to one for all classes?

Reply:
Ok, I'll stick to `_initAttributes` in this PR, and change other existing seprated instance attribute in seprated PR.